packages/DataGrid/src/DataGrid.js

```diff
@@ -0,0 +1,628 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { VariableSizeGrid as Grid } from "react-window";
+import extractChildren from "@paprika/helpers/lib/extractChildren";
+import Spinner from "@paprika/spinner";
+import Bubbly from "@paprika/icon/lib/BubblySearchNoResults";
+import Cell from "./components/Cell";
+import useGridEventHandler from "./hooks/useGridEventHandler";
+import ColumnDefinition from "./components/ColumnDefinition";
+import * as styled from "./DataGrid.styles";
+import WhenScrollBarReachedBottom, { End } from "./components/WhenScrollBarReachedBottom";
+import InfinityScroll from "./components/InfinityScroll";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  data: PropTypes.arrayOf(PropTypes.shape({})),
+  hasNoRecords: PropTypes.bool,
+  height: PropTypes.number,
+  isIdle: PropTypes.bool,
+  onClick: PropTypes.func,
+  onEnter: PropTypes.func,
+  onKeyDown: PropTypes.func,
+  onRowChecked: PropTypes.func,
+  onShiftSpaceBar: PropTypes.func,
+  onSpaceBar: PropTypes.func,
+  rowHeight: PropTypes.number,
+  width: PropTypes.number,
+};
+const defaultProps = {
+  data: [],
+  hasNoRecords: false,
+  height: 600,
+  isIdle: false,
+  onClick: null,
+  onEnter: null,
+  onKeyDown: () => {},
+  onRowChecked: () => {},
+  onShiftSpaceBar: null,
+  onSpaceBar: null,
```

Comment:
Our standard is `onPressXxx` not `onXxx` (`onEnter`, too)

Reply:
`onPressShifSpaceBar` `onPressSpaceBar` 👍